static/js/util/test_utils.js

```diff
@@ -1,7 +1,36 @@
 import TestUtils from 'react-addons-test-utils';
+import { assert } from 'chai';
 
 export const modifyTextField = (field, text) => {
   field.value = text;
   TestUtils.Simulate.change(field);
   TestUtils.Simulate.keyDown(field, {key: "Enter", keyCode: 13, which: 13});
 };
+
+export const isActiveDialog = (dialog) => (
+  dialog.style["left"] === "0px"
+);
+
+let findActiveDialog = (dialogClassName) => (
+  [...document.getElementsByClassName(dialogClassName)].find(dialog => (
+    isActiveDialog(dialog)
+  ))
+);
+
+export const noActiveDialogs = (dialogClassName) => {
+  return findActiveDialog(dialogClassName) === undefined;
+};
+
+export const activeDialog = (dialogClassName) => {
+  let dialog = findActiveDialog(dialogClassName);
+  assert.isDefined(dialog, `dialog element w/ className '${dialogClassName}' should be active`);
+  return dialog;
+};
+
+export const activeDeleteDialog = () => {
+  return activeDialog('deletion-confirmation');
+};
+
+export const noActiveDeleteDialogs = () => {
+  return noActiveDialogs('deletion-confirmation');
+};
```

Comment:
the one-line functions you could skip the return keyword by using parentheses if you want, btw